docker/.editorconfig

```diff
@@ -3,10 +3,15 @@ root = true
 [*]
 end_of_line = lf
 insert_final_newline = true
-
-[*]
+indent_style = space
```

Comment:
While I am all up for standardizing, I am not sure if I like these kinds of files being in the repo. Curious to hear the opinion of others on this matter.